src/katt_callbacks.erl

```diff
@@ -263,7 +265,7 @@ http_request( #katt_request{ method = Method
   Hdrs1 = proplists:delete("x-katt-request-sleep", Hdrs0),
   Hdrs = proplists:delete("x-katt-request-timeout", Hdrs1),
   timer:sleep(Sleep),
-  katt_util:external_http_request(Url, Method, Hdrs, Body, Timeout, []).
+  katt_util:external_http_request(Url, Method, Hdrs, Body, Timeout, Params).
```

Comment:
send only the hackney_options as HackneyOpts, not the entire katt params to keep katt_util:external_http_request minimalistic